recipes/genespeak/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "genespeak" %}
+{% set version = "0.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/genespeak-{{ version }}.tar.gz
+  sha256: 40f4945229b68b941b38bf53875497fc9d08305588bb8b521eadcf08380e48f0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - genespeak
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sugatoray/genespeak/
```

Comment:
Cool package! This URL is 404ing for me though :/

Reply:
Thanks. I haven't made the github repo public yet. Some more work before I open up the repository. So, it should show a 404 for now. But the doc_url should work (although it's also work in progress).

Reply:
@jaimergp Now the repository is public.